pkg/test/masteruserrecord/assertion.go

```diff
@@ -181,6 +183,7 @@ TierNamespaces:
 		assert.Nil(a.t, userAccount.Spec.NSTemplateSet.ClusterResources)
 	} else {
 		assert.Equal(a.t, tier.Spec.ClusterResources.Revision, userAccount.Spec.NSTemplateSet.ClusterResources.Revision)
+		assert.Equal(a.t, strings.ToLower(fmt.Sprintf("%s-clusterresources-%s", tier.Name, tier.Spec.ClusterResources.Revision)), tier.Spec.ClusterResources.TemplateRef)
```

Comment:
the same here. In addition, the third parameter should be the actual value set in UserAccount/NSTemplateSet
```suggestion
		assert.Equal(a.t, tier.Spec.ClusterResources.TemplateRef, userAccount.Spec.NSTemplateSet.ClusterResources.TemplateRef)
```